gui/src/2D/controls/image.ts

```diff
@@ -511,7 +511,12 @@ export class Image extends Control {
             value = this._svgCheck(value);
         }
 
-        this._domImage = document.createElement("img");
+        // Should abstract platform instead of using LastCreatedEngine
+        const engine = Engine.LastCreatedEngine;

```

Comment:
Same I wonder if it should be along those lines:

this._host?.getScene()?.getEngine() || Engine.LastCreatedEngine; ?